stringio.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |s|
   s.name = "stringio"
   s.version = source_version
 
-  s.required_rubygems_version = Gem::Requirement.new(">= 2.6")
+  s.required_rubygems_version = Gem::Requirement.new(">= 3.0")
```

Comment:
Why do we need this?

We may be able to remove `s.required_rubygems_version` entirely.

Reply:
Can `s.required_rubygems_version` be omitted?

Reply:
Yes.
For example, csv doesn't specify it: https://github.com/ruby/csv/blob/master/csv.gemspec

Reply:
Basically, It's not required because we have no incompatibility of `Gem::Specification` today.